src/components/MobileNavigation/MobileNavigation.js

```diff
@@ -12,6 +13,12 @@ function MobileNavigation(props) {
     onClose: null,
   });
 
+  const VercelStyle = styled.a`
+    display: block;
+    width: 10px;
+    color: #f90606;
```

Comment:
Is this color necessary? If not, can it be removed?

Also, I think we could use some space above and below the banner. Maybe we can do a swap?

```suggestion
    margin: 20px 0 10px;
```